### PR TITLE
Implement UI for all validators page

### LINKED ISSUE #745: Implement UI for all validators page

### PR DESCRIPTION
**Pull request by Benzbeeb**

### CODE REVIEW

**Comment by evilpeach** [comment]
@Benzbeeb Please fix a conflict.

File: scan/index.html [hunk 1]
```diff
@@ -1,29 +1,26 @@
...
-    <title>
-      D3N Explorer - Block Explorer for Decentralized Data Delivery Network,
-      Powered by Band Protocol
-    </title>
-  </head>
-  <body>
-    <div id="root"></div>
-    <script src="./src/Index.re"></script>
-  </body>
+<head>
```
Comment (taobun) [inline scan/index.html]: What?

File: scan/src/reusable/InfoHL.re [hunk 2]
```diff
@@ -125,7 +135,24 @@ let make = (~info, ~header, ~isLeft=true) => {
...
+     | Fraction(x, y, space) =>
+       <Text
+         value={(x |> Format.iPretty) ++ (space ? " / " : "/") ++ (y |> Format.iPretty)}
+         size=Text.Lg
+         weight=Text.Semibold
+         spacing={Text.Em(0.02)}
+         code=true
+       />
+
+     | Address(address, textColor) =>
```
Comment (taobun) [inline scan/src/reusable/InfoHL.re]: unused testColor ?

**Review by taobun — CHANGES_REQUESTED** [review]

**Review by taobun — APPROVED** [review]
LGTM

**Review by evilpeach — APPROVED** [review]

**Pull request merged**